trade-finance-manager-ui/templates/case/underwriting/managers-decision/_macros/managers-decision-submitted.njk

```diff
@@ -45,7 +45,7 @@
 
             <h4 class="govuk-heading-m govuk-!-font-weight-bold" data-cy="comments-heading">{{commentsHeadingText }} </h4>
 
-            <p class="ukef-preserve-white-space" data-cy="conditions-value">{{ decision.comments | safe }}</p>
+            <p class="ukef-preserve-white-space" data-cy="conditions-value">{{ decision.comments }}</p>
```

Comment:
We use `safe` in quite a lot of other places in DTFS. Is there anything in the backlog about removing them?

I get the impression it's only supposed to be used when you're absolutely sure that what you're displaying can't be malicious, which we can't be sure of when it's from a user input.

If there isn't already a ticket for it, I think we should raise one to remove all the instances of `safe`. What do you think?

Reply:
`safe` should have only been used where we do not want to escape HTML tags, in which case ` | safe` NJK filter should be removed please.

Reply:
Agreed. We need to remove the safe pipe. A separate ticket will be needed for it